libs/deer/tests/test_impls_core_time.rs

```diff
@@ -0,0 +1,17 @@
+use core::time::Duration;
+
+use approx::assert_relative_eq;
+use deer_desert::{assert_tokens_with_assertion, Token};
+use proptest::prelude::*;
+
+#[cfg(not(miri))]
+proptest! {
+    #[test]
+    fn duration_ok(value in any::<Duration>()) {
+        let input = value.as_secs_f64();
+
+        assert_tokens_with_assertion(|received: Duration| {
+            assert_relative_eq!(received.as_secs_f64(), value.as_secs_f64());
```

Comment:
Is `approx` used in future implementations? Or can we simply do `a - b < f64:EPSILON` for f64 comparisons?

Reply:
nope not at all, we can indeed just use `a - b < f64:EPSILON`. Good catch!

Reply:
fixed in [9048070](https://github.com/hashintel/hash/pull/2390/commits/904807051ec402a73d550893bb9cfd1aa33cbc7b)